### PR TITLE
Hanlon's Razor

### PR DESCRIPTION
**Pull request by DillonAd**
- Adding section for Hanlon's Razor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +91,14 @@ See also:
 
 - [The Spotify Model](#the-spotify-model)
 
+### Hanlon's Razor
+
+[Hanlon's Razor on Wikipedia](https://en.wikipedia.org/wiki/Hanlon%27s_razor)
+
+> "Never attribute to malice that which is adequately explained by stupidity."
+
+This principal suggests that actions resulting in a negative outcome were not a result of ill will. Instead the negative outcome is more likely attributed to those actions and/or the impact being not fully understood.
```
Comment (dwmkerr) [inline README.md]: `principle` I think rather than `principal`

Comment (DillonAd) [reply README.md]: You are right. I have fixed the typo.

**Comment by DillonAd** [comment]
Added the author to the quote and fixed the typo as suggested!

**Comment by dwmkerr** [comment]
Excellent, just let me know your twitter handle if you have one and want me to mention you when I share the update!

**Comment by DillonAd** [comment]
Looks like you already found it! Thanks!

**Comment by dwmkerr** [comment]
No worries, thanks for this one Dillon!

**Pull request merged**